lib/improver/nowcasting/pysteps_advection.py

```diff
@@ -50,7 +55,7 @@ class PystepsExtrapolate(object):
         https://pysteps.readthedocs.io/en/latest/generated/
         pysteps.extrapolation.semilagrangian.extrapolate.html
     """
-    def __init__(self, interval, max_lead_time):
+    def __init__(self, interval, max_lead_time, attributes_dict=None):
```

Comment:
Attributes dict I would usually pass into `process`.  This makes more sense in this particular instance as well, as `process` is where we're passing in anything specific to the outputs (as opposed to what the plugin does).  Then there's no need to make it a class member.

Reply:
Moved.